app/javascript/components/FacetsAccordion.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Accordion from 'react-bootstrap/Accordion';
+import Card from 'react-bootstrap/Card';
+
+import Filters from './Filters';
+
+export default function FacetsAccordion(props) {
+  return (
+    // Accordions provide a way to restrict Card components to only open one at a time.
+    // https://react-bootstrap.github.io/components/accordion/
```

Comment:
For our code based are we supposed to mention where we get each component from if its not apart of the standard library?

Reply:
I think it's useful in places like this one where it's non-obvious

Reply:
That summarizes my thinking.  My rationale here was to link for maintainers' convenience.